lib/common/xml.c

```diff
@@ -843,6 +843,12 @@ pcmk__xe_set_content(xmlNode *node, const char *format, ...)
             va_end(ap);
         }
 
+        /* xmlNodeSetContent() frees node->children and replaces it with new
+         * text. Free it explicitly to ensure private data gets freed after we
+         * drop xmlDeregisterNodeDefault().
```

Comment:
it is dropped :)

Reply:
I meant to remove that last clause in the later commit and forgot